test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -151,14 +156,24 @@ def test_ec2_pytorch_inference_cpu(pytorch_inference, ec2_connection, region, cp
 
 @pytest.mark.usefixtures("sagemaker")
 @pytest.mark.model("densenet")
-@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GRAVITON_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_GRAVITON_INSTANCE_TYPE, indirect=True)
 @pytest.mark.parametrize("ec2_instance_ami", [test_utils.UL20_CPU_ARM64_US_WEST_2], indirect=True)
 def test_ec2_pytorch_inference_graviton_cpu(
     pytorch_inference_graviton, ec2_connection, region, cpu_only
 ):
     ec2_pytorch_inference(pytorch_inference_graviton, "graviton", ec2_connection, region)
 
 
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("densenet")
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_GRAVITON_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ec2_instance_ami", [test_utils.UL20_CPU_ARM64_US_WEST_2], indirect=True)
+def test_ec2_pytorch_inference_graviton_gpu(
+    pytorch_inference_graviton, ec2_connection, region, gpu_only
+):
+    ec2_pytorch_inference(pytorch_inference_graviton, "graviton", ec2_connection, region)
```

Comment:
This does not test whether the GPU was engaged or not. We need to add further checks to make sure the model was not compiled for the CPU.